`make generate-openapi-spec` fails

@Moopli and I can reproduce this error in our local environments:

```
~/dev/hyperledger/aries-framework-go$ make all
Running scripts/check_license.sh
Examining working directory changes
All files have SPDX-License-Identifier headers
Running scripts/check_lint.sh
rm -f coverage.txt
rm -Rf ./build
Generating and validating controller API specifications using Open API
Generating Open API spec
Validating generated spec
2019/09/20 19:46:43 
The swagger spec at "build/rest/openapi/spec/openAPI.yml" showed up some valid but possibly unwanted constructs.
2019/09/20 19:46:43 See warnings below:
2019/09/20 19:46:43 - WARNING: spec has no valid path defined

The swagger spec at "build/rest/openapi/spec/openAPI.yml" is invalid against swagger specification 2.0.
See errors below:
- .info in body is required

Makefile:38: recipe for target 'generate-openapi-spec' failed
make: *** [generate-openapi-spec] Error 1
```

Contents of the `openAPI.yml` file:

```
$ cat build/rest/openapi/spec/openAPI.yml 
paths: {}
swagger: "2.0"
$ 
```

--- a/Makefile
+++ b/Makefile
@@ -5,12 +5,14 @@
 GO_CMD ?= go
 # Controller API entry point to be used for generating Open API specifications
 OPENAPI_SPEC_META=cmd/aries-agentd/main.go
+OPENAPI_DOCKER_IMG=quay.io/goswagger/swagger
+OPENAPI_DOCKER_IMG_VERSION=v0.20.1
 
 .PHONY: all
 all: checks generate-openapi-spec unit-test
 
 .PHONY: checks
-checks: license lint
+checks: license lint generate-openapi-spec
 
 .PHONY: lint
 lint:
@@ -37,4 +39,6 @@ clean:
 generate-openapi-spec: clean
 	@echo "Generating and validating controller API specifications using Open API"
 	@mkdir -p build/rest/openapi/spec
-	@SPEC_META=$(OPENAPI_SPEC_META) SPEC_LOC=build/rest/openapi/spec scripts/generate-openapi-spec.sh
+	@SPEC_META=$(OPENAPI_SPEC_META) SPEC_LOC=build/rest/openapi/spec  \
+	DOCKER_IMAGE=$(OPENAPI_DOCKER_IMG) DOCKER_IMAGE_VERSION=$(OPENAPI_DOCKER_IMG_VERSION)  \
+	scripts/generate-openapi-spec.sh

--- a/scripts/generate-openapi-spec.sh
+++ b/scripts/generate-openapi-spec.sh
@@ -10,13 +10,15 @@ CODEBASE="$(dirname "$PWD")"
 SPEC_LOC="${SPEC_LOC}"
 SPEC_META="${SPEC_META:-cmd/aries-agentd/main.go}"
 OUTPUT="$SPEC_LOC/openAPI.yml"
+IMAGE="${DOCKER_IMAGE:-quay.io/goswagger/swagger}"
+IMAGE_VERSION="${DOCKER_IMAGE_VERSION:-latest}"
 
 # generate and validate commands
 GENERATE_COMMAND="generate spec $SPEC_META -o $OUTPUT"
 VALIDATE_COMMAND="validate $OUTPUT"
 
 echo "Generating Open API spec"
-docker run --rm -it -e GOPATH=$HOME/go:/go -v $HOME:$HOME -w $(pwd) quay.io/goswagger/swagger $GENERATE_COMMAND
+docker run --rm -e GOPATH=$HOME/go:/go -v $HOME:$HOME -w $(pwd) ${IMAGE}:${IMAGE_VERSION} $GENERATE_COMMAND
 
 echo "Validating generated spec"
-docker run --rm -it -e GOPATH=$HOME/go:/go -v $HOME:$HOME -w $(pwd) quay.io/goswagger/swagger $VALIDATE_COMMAND
+docker run --rm -e GOPATH=$HOME/go:/go -v $HOME:$HOME -w $(pwd) ${IMAGE}:${IMAGE_VERSION} $VALIDATE_COMMAND

